### PR TITLE
Alex krasn/docs update readme keyboard shortcuts

### PR DESCRIPTION
**Pull request by alex-krasn**
Updated README.md and added icon picture keyboard-shortcuts-icon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,186 +1,188 @@
...
+
+When you want to resume your project, you first need to create a connection to the same blob storage container. Repeat the steps above to do this. Then, go to the application settings page (gear icon) and see if your project's security token is there. If it isn't, add a new security token and copy over your token name and key from the previous step. Then click Save Settings.
+
+##### Resume a project #####
+
+Finally, go to the main page (house icon) and click Open Cloud Project. Then select the blob storage connection, and select your project's .proj file. The application will load all of the project's settings because it has the security token.
+
+#### Keyboard Shortcuts and useful tips ####
+
+Labeling tool allows a number of keyboard shortcuts to support accessibility and also sometimes make labeling easier and faster. You can find them in the right top corner (icon with book and letter "i" ![alt text](docs/images/keyboard-shortcuts-icon.png "Keyboard shortcuts menu icon")).
```
Comment (stew-ro) [inline README.md:162]: is this the only change?

Comment (alex-krasn) [reply README.md:162]: yes

File: README.md [hunk 2]
```diff
@@ -171,6 +171,8 @@ Hotkeys of 1 through 0 and all letters are assigned to first 36 tags, after you
 
 Hold Alt key and click on tag name, user can change the tag's name.
 
+To delete all labels for a tag - select all labels for a tag on document then press 'delete' key.
```
Comment (alex-krasn) [inline README.md:174]: @stew-ro 
and this

File: README.md [hunk 3]
```diff
@@ -157,9 +157,9 @@ When you want to resume your project, you first need to create a connection to t
 
 Finally, go to the main page (house icon) and click Open Cloud Project. Then select the blob storage connection, and select your project's .proj file. The application will load all of the project's settings because it has the security token.
 
-#### Keyboard Shortcuts ####
+#### Keyboard Shortcuts and useful tips ####
 
-Labeling tool allows a number of keyboard shortcuts to support accessibility and also sometimes make labeling easier and faster.
+Labeling tool allows a number of keyboard shortcuts to support accessibility and also sometimes make labeling easier and faster. You can find them in the right top corner (icon with book and letter "i" ![alt text](docs/images/keyboard-shortcuts-icon.png "Keyboard shortcuts menu icon")).
```
Comment (stew-ro) [inline README.md:162]: The image isn't showing up for me in rich diff

Comment (alex-krasn) [reply README.md:162]: For me either. But it does on the branch itself https://github.com/microsoft/OCR-Form-Tools/tree/alex-krasn/docs-update-README-keyboard-shortcuts

Comment (stew-ro) [reply README.md:162]: Can we move the image to a new line? 
image

maybe something like this:
```
You can view them by clicking the following icon on the right side of the title bar:
![alt text](docs/images/keyboard-shortcuts-icon.png "Keyboard shortcuts menu icon")
```

**Pull request merged**